packages/app-project/src/screens/ClassifyPage/components/YourProjectStats/YourProjectStatsContainer.js

```diff
@@ -0,0 +1,42 @@
+import { useContext } from 'react'
+import { MobXProviderContext, observer } from 'mobx-react'
+
+import useYourProjectStats from './useYourProjectStats.js'
+import YourProjectStats from './YourProjectStats.js'
+
+function storeMapper(store) {
+  const { project, user } = store
+
+  return {
+    projectID: project.id,
+    userID: user.id,
+    userLogin: user.login
+  }
+}
+
+/**
+ * This is a relatively simple container for ProjectStats, but data fetching
+ * and store observing are purposely separated from the presentational component
+ * styling and logic. Fetching user data requires authorization, making it
+ * complicated to use a mock library like MSW for useYourProjectStats() hook.
+ */
```

Comment:
Just noting I considered using MSW in order to see YourProjectStatsContainer in Storybook and unit tests, but mocking authorized requests for user stats was too challenging for the return.

Reply:
If it's of any use, the `SubjectPicker` stories have an example of mocking an authenticated request to Panoptes with MSW.

Reply:
You don't need a signed-in user to make requests in the SubjectPicker do you? 🤔 ERAS user stats requests require authorization from the signed-in user. User authorization is what I had a difficult time mocking for YourProjectStatsContainer.

Reply:
The subject picker makes an auth'ed request to get the already seen and status of each subject for your user account. I don't know if this will help here, but in the subject picker you can ignore the auth headers and mock the endpoint directly. This is the auth'ed API query:
https://github.com/zooniverse/front-end-monorepo/blob/1862adebef4ddfde8d071669c99038bf3343383c/packages/app-project/src/shared/components/SubjectPicker/SubjectPicker.stories.js#L41-L49

Reply:
Gotcha, that makes much more sense! In YourProjectStatsContainer I started mocking all the requests in `useYourProjecStats()` hook, got to the authorization part, and said 'uh oh' better just separate the presentational component into its own stories. I'm going to leave this PR as is, but definitely want to use MSW more in the future.

Reply:
TBH, if your container passes the ERAS response directly into the presentation component, then you're already testing with mock API data anyway. The subject picker transforms the API response, so mocking tests that the transformation is working.